linear-operators/introduction-to-linear-operators/introduction-to-linear-operators.jmd

```diff
@@ -0,0 +1,386 @@
+---
+title: Introduction to LinearOperators.jl
+author: Abel Soares Siqueira
+---
+
+![](../../jso-banner.png)
+
+```julia; results="hidden"
+using Pkg
+pkg"activate ."
+pkg"add LinearOperators"
+pkg"add FFTW"
+pkg"add Plots"
+pkg"add PyPlot"
+pkg"add Weave"
+pkg"instantiate"
+pkg"status"
+```
+
+Julia version and JSO packages:
+```julia; echo=false
+pkgs = ["LinearOperators"]
+
+using Pkg
+ctx=Pkg.Types.Context()
+display("text/html", "<img src=\"https://img.shields.io/badge/julia-$VERSION-3a5fcc.svg?style=flat-square&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAOCAMAAAAolt3jAAAB+FBMVEUAAAA3lyU3liQ4mCY4mCY3lyU4lyY1liM3mCUskhlSpkIvkx0zlSEeigo5mSc8mio0liKPxYQ/nC5NozxQpUBHoDY3lyQ5mCc3lyY6mSg3lyVPpD9frVBgrVFZqUpEnjNgrVE3lyU8mio8mipWqEZhrVJgrVFfrE9JoTkAVAA3lyXJOjPMPjNZhCowmiNOoz1erE9grVFYqUhCnjFmk2KFYpqUV7KTWLDKOjK8CADORj7GJx3SJyVAmCtKojpOoz1DnzFVeVWVSLj///+UV7GVWbK8GBjPTEPMQTjPTUXQUkrQSEGZUycXmg+WXbKfZ7qgarqbYraSVLCUV7HLPDTKNy7QUEjUYVrVY1zTXFXPRz2UVLmha7upeMCqecGlcb6aYLWfaLrLPjXLPjXSWFDVZF3VY1zVYlvRTkSaWKqlcr6qesGqecGpd8CdZbjo2+7LPTTKOS/QUUnVYlvVY1zUXVbPSD6TV7OibLuqecGqecGmc76aYbaibLvKOC/SWlPMQjrQUEjRVEzPS0PLPDL7WROZX7WgarqibLucY7eTVrCVWLLLOzLGLCLQT0bIMynKOC7FJx3MPjV/Vc+odsCRUa+SVLCDPaWVWLKWWrLJOzPHOTLKPDPLPDPLOzLLPDOUV6+UV7CVWLKVWLKUV7GUWLGPUqv///8iGqajAAAAp3RSTlMAAAAAAAAAAAAAABAZBAAAAABOx9uVFQAAAB/Y////eQAAADv0////pgEAAAAAGtD///9uAAAAAAAAAAcOQbPLfxgNAAAAAAA5sMyGGg1Ht8p6CwAAFMf///94H9j///xiAAAw7////65K+f///5gAABjQ////gibg////bAAAAEfD3JwaAFfK2o0RAAAAAA4aBQAAABEZAwAAAAAAAAAAAAAAAAAAAIvMfRYAAAA6SURBVAjXtcexEUBAFAXAfTM/IDH6uAbUqkItyAQYR26zDeS0UxieBvPVbArjXd9GS295raa/Gmu/A7zfBRgv03cCAAAAAElFTkSuQmCC\">")
+for p in pkgs
+  uuid=ctx.env.project.deps[p]
+  v=ctx.env.manifest[uuid].version
+  c=string(hash(p) % 0x1000000, base=16)
+  display("text/html", "<img src=\"https://img.shields.io/badge/$p-$v-brightgreen?color=$c\">")
+end
+println("Tutorial beginning:")
```

Comment:
Kind of. I included this `println` because the block prints something. It could be printing simply a `nothing`, for instance. If I use `display` I can create a title, but then it will repeat the title because that'll be the last thing in the block.
This can be changed to a "please ignore this line".

Reply:
Maybe it could be a `pkg> status`?

Reply:
This is much better, thanks.